src/components/Breadcrumb/Breadcrumb.less

```diff
@@ -0,0 +1,43 @@
+.lucid-Breadcrumb {
```

Comment:
not sure why you would avoid it, the compiler just flattens it out anyways. besides we're shouldn't be optimizing for best compile time.